testsuite/openshift/objects/auth_config/sections.py

```diff
@@ -180,35 +197,41 @@ def add_uma(self, name, endpoint, credentials_secret, **common_features):
 
 
 class ResponseSection(Section):
-    """Section which contains response configuration"""
+    """Section which contains response configuration. todo envoyDynamicMetadata"""
+
+    @property
+    def success_headers(self):
+        """Nested dict for most of the section."""
+        return self.section.setdefault("success", {}).setdefault("headers", {})
+
+    @modify
+    def clear_success_headers(self):
+        """Clears content of a success headers"""
+        self.success_headers.clear()
 
     def _add(
         self,
         name: str,
         value: dict,
-        wrapper_key: str = None,
-        wrapper: Literal["httpHeader", "envoyDynamicMetadata"] = None,
-        **common_features
+        **common_features,
     ):
         """Add response to AuthConfig"""
-        if wrapper:
-            value["wrapper"] = wrapper
-        if wrapper_key:
-            value["wrapperKey"] = wrapper_key
-
-        self.add_item(name, value, **common_features)
+        add_common_features(value, **common_features)
+        self.success_headers.update({name: value})
```

Comment:
are these `success_headers` used somewhere in the testsuite?

Reply:
Nope. Just in the `_add` function.